llvm/lib/SYCLLowerIR/ModuleSplitter.cpp

```diff
@@ -1191,7 +1200,7 @@ splitSYCLModule(std::unique_ptr<Module> M, ModuleSplitterSettings Settings) {
   ModuleDesc MD = std::move(M); // makeModuleDesc() ?
   // FIXME: false arguments are temporary for now.
   auto Splitter =
-      getDeviceCodeSplitter(std::move(MD), Settings.Mode, false, false);
+      getDeviceCodeSplitter(std::move(MD), Settings.Mode, false, false, false);
```

Comment:
could we add comments for what all the bool arguments are here? like
```
/*foo=*/false
```

Reply:
Added comments for bool args.